contract_manager/.eslintrc.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+  root: true,
+  parser: "@typescript-eslint/parser",
+  plugins: ["@typescript-eslint"],
+  extends: ["eslint:recommended", "plugin:@typescript-eslint/recommended"],
+  rules: {
+    "@typescript-eslint/no-explicit-any": "off",
```

Comment:
Can't we instead turn it off for the only place that it's defined? In general it's good to keep this enabled by default?

Reply:
We had ~40 warnings, fixed most and ignored 4 of them